T/Torch/build_tarballs.jl

```diff
@@ -232,37 +226,39 @@ for p in cuda_platforms
 end
 
 platforms = expand_cxxstring_abis(platforms)
+accelerate_platforms = expand_cxxstring_abis(accelerate_platforms)
 mkl_platforms = expand_cxxstring_abis(mkl_platforms)
-blis_platforms = expand_cxxstring_abis(blis_platforms)
 openblas_platforms = expand_cxxstring_abis(openblas_platforms)
 cuda_platforms = expand_cxxstring_abis(cuda_platforms)
 
 # The products that we will ensure are always built
 products = [
+    FileProduct("share/ATen/Declarations.yaml", :declarations_yaml),
     LibraryProduct(["libtorch", "torch"], :libtorch),
     LibraryProduct(["libtorch_cpu", "torch_cpu"], :libtorch_cpu),
 ]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae")),
-    Dependency("blis_jll"; platforms = blis_platforms),
     Dependency("CPUInfo_jll"; compat = "0.0.20201217"),
     Dependency("CUDNN_jll", v"8.2.4"; compat = "8", platforms = cuda_platforms),
     Dependency("Gloo_jll";  compat = "0.0.20210521", platforms = filter(p -> nbits(p) == 64, platforms)),
     Dependency("LAPACK_jll"; platforms = openblas_platforms),
-    Dependency("MKL_jll"; platforms = mkl_platforms),
-    BuildDependency("MKL_Headers_jll"; platforms = mkl_platforms),
-    Dependency("OpenBLAS_jll"; platforms = openblas_platforms),
+    # Dependency("MKL_jll"; platforms = mkl_platforms), # MKL is avoided for all platforms
+    # BuildDependency("MKL_Headers_jll"; platforms = mkl_platforms), # MKL is avoided for all platforms
+    Dependency("OpenBLAS32_jll"; platforms = openblas_platforms),
     Dependency("PThreadPool_jll"; compat = "0.0.20210414"),
     Dependency("SLEEF_jll", v"3.5.2"; compat = "3"),
-#    Dependency("TensorRT_jll"; platforms = cuda_platforms), # Building with TensorRT is not supported: https://github.com/pytorch/pytorch/issues/60228
+    # Dependency("TensorRT_jll"; platforms = cuda_platforms), # Building with TensorRT is not supported: https://github.com/pytorch/pytorch/issues/60228
     Dependency("XNNPACK_jll"; compat = "0.0.20210622"),
     Dependency(PackageSpec("protoc_jll", Base.UUID("c7845625-083e-5bbe-8504-b32d602b7110")); compat="~3.13.0"),
     HostBuildDependency(PackageSpec("protoc_jll", Base.UUID("c7845625-083e-5bbe-8504-b32d602b7110"), v"3.13.0")),
+    RuntimeDependency("CUDA_Runtime_jll"),
```

Comment:
Was this supposed to be a dependency for all platforms?  Or only the CUDA ones?

Reply:
Its required, I believe, by the CUDA augment block that I added - I guess that is necessary in order to be able to get hold of the CUDA artifacts.